Parsers/Calculator.js

```diff
@@ -0,0 +1,38 @@
+/*
+activation_example: !calculate <expression>
+regex: !calculate (.+)
```

Comment:
key:value pairs in the header template are affected by leading spaces.

Reply:
I have removed the spaces. 
Thanks